dataikuapi/dssclient.py

```diff
@@ -95,14 +95,14 @@ def list_running_notebooks(self, as_objects=True):
         """
         List the currently-running Jupyter notebooks
 
-        :param boolean as_objects: if True, each returned item will be a :class:`dataikuapi.dss.notebook.DSSNotebook`
+        :param boolean as_objects: if True, each returned item will be a :class:`dataikuapi.dss.notebook.DSSJupyterNotebook`
```

Comment:
This part is still a mix between the new and the old. We should have 2 different methods:
- list_running_notebooks() => the old one, returning DSSNotebook objects. Marked as deprecated
- list_jupyter_notebooks(active) => the new one, returning DSSJupyterNotebook objects

Reply:
Done. Tell me if you like it.

Reply:
Looks good now.